### PR TITLE
Support RDS BackgroundColor in SlackSheet

### PR DESCRIPTION
**Pull request by benisgold**
Fixes APP-####

## What changed (plus any additional context for devs)
- now you can use regular hex colors or RDS `BackgroundColor`s in the SlackSheet `backgroundColor` prop

## Screen recordings / screenshots
https://www.loom.com/share/fb606fe59a784df8a6b5d6d99d7a969d

### CODE REVIEW

File: src/components/sheet/SlackSheet.js [hunk 1]
```diff
@@ -156,7 +159,25 @@ export default forwardRef(function SlackSheet(
     yPosition.value = event.contentOffset.y;
   });
 
-  const bg = backgroundColor || colors.white;
+  const { backgroundColors, colorMode } = useContext(ColorModeContext);
+  const accentColorContextValue = useContext(AccentColorContext);
+
+  let bg = backgroundColor;
```
Comment (skylarbarrera) [inline src/components/sheet/SlackSheet.js:165]: I think this would read better 
`
bg = colors.white
if (backgroundColor){
bg = backgroundColor
}
`

Comment (skylarbarrera) [reply src/components/sheet/SlackSheet.js]: ig this would need to check that its not 'accent' as well

**Review by skylarbarrera — APPROVED** [review]
im fine with this but the logic seems kinda funky. im not sure if this is worth doing here vs just passing in the color value separately -> or even making a separate RDS sheet

would love @estrattonbailey or @jkadamczyk 's thoughts on this

**Comment by benisgold** [comment]
@skylarbarrera i just copy pasted the logic directly from RDS `BackgroundProvider`

about your 2 suggestions:
- extracting the color value from a `BackgroundColor` is impossible with current RDS implementation. i could make a legitimate way to extract the value like `useForegroundColor` but idk if there would be much a use case besides this specific `SlackSheet` scenario. to me it seems like unneeded effort
- Making a separate RDS sheet would be ideal but there are so many things wrong with SlackSheet that doing it properly would be a huge undertaking. Regardless, I still think we should do it at some point but I just don't want to right now haha

**Comment by benisgold** [comment]
@skylarbarrera jk im dumb, closing this

**Pull request closed without merging**